packages/lib/gpcircuits/circuits/numeric-value.circom

```diff
@@ -38,11 +38,11 @@ template NumericValueModule() {
     // preserving order.
     var ABS_POD_INT_MIN = 1 << 63;
     
-    // Check that minValue <= numericValue <= maxValue.
-    signal boundsCheck <== BoundsCheckModule(64)(
+    // Check that minValue <= numericValue <= maxValue and return the
+    // result.
+    signal output boundsCheck <== BoundsCheckModule(64)(
```

Comment:
Maybe call this `isInBounds` so it's clearer that it's a boolean and what it means?
Could've been applicable before too, but it occurs to me more strongly now that this is an output, so part of the API of the module.